dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetServiceBean.java

```diff
@@ -260,19 +260,17 @@ public PasswordChangeAttemptResponse attemptPasswordReset(BuiltinUser user, Stri
             AuthenticatedUser authUser = authService.getAuthenticatedUser(user.getUserName());
 
             String toAddress = authUser.getEmail();
-            String subject = "Dataverse Password Reset Successfully Changed";
+            String subject = BundleUtil.getStringFromBundleWithLocale("notification.email.passwordResetConfirmation.subject", authUser.getNotificationsLanguage());
 
 
-            String messageBody = "Hi " + authUser.getName() + ",\n\n"
-                    + "Your Dataverse account password was successfully changed.\n\n"
-                    + "Please contact us if you did not request this password reset or need further help.\n\n";
+            String messageBody = BundleUtil.getStringFromBundleWithLocale("notification.email.passwordResetConfirmation.subject", authUser.getNotificationsLanguage(), authUser.getName(), user.getUserName());
```

Comment:
`... .subject`? Did we mean `notification.email.passwordResetConfirmation`?

Reply:
sure